### PR TITLE
docs(ipfs): review api/cli section MTA-3423

### PR DESCRIPTION
**Pull request by farhakennar**

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

**Review by jcirinosclwy — APPROVED** [review]

File: labs/ipfs/api-cli/pin-operations.mdx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+Alternatively, you can also run the following API command. Just make sure to replace the example values with your own, using the API configuration table below:
+
+```
+  curl -X POST \
+    -H "X-Auth-Token: $SCW_SECRET_KEY" \
+    -H "Content-Type: application/json" \
+    -d '{
+      "name": "MyNewPinByURL",
+      "url": "https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png",
+      "volume_id": "6170692e-7363-616c-6577-61792e636f6d"
```
Comment (OnsagerHe) [inline labs/ipfs/api-cli/pin-operations.mdx:122]: replace volume uuid by example value: 11111111-1111-1111-1111-111111111111

File: labs/ipfs/api-cli/pin-operations.mdx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+Cid.Name        value:"MyNewPinByCID"
+```
+
+<Message type="note">
+  If the content is successfully pinned, the response will provide the CID of your content pointed by the URL. If it fails, no CID will be returned. You will still be able to retrieve the URL of the pinned content so you can retry.
+</Message>
+
+## Deleting a pin
+
+This operation allows you to delete a given pin within a volume using its pin ID. If you were the last owner of the associated content, the content will be permanently deleted.
```
Comment (OnsagerHe) [inline labs/ipfs/api-cli/pin-operations.mdx:154]: `If you were the last owner of the associated content` this sentence can be deleted as it explains an implementation detail that is not important.

**Review by OnsagerHe — CHANGES_REQUESTED** [review]

File: labs/ipfs/api-cli/pin-operations.mdx (hunk 1 above)
Comment (bene2k1) [reply labs/ipfs/api-cli/pin-operations.mdx:122]:
```suggestion
      "volume_id": "11111111-1111-1111-1111-111111111111"
```

File: labs/ipfs/api-cli/pin-operations.mdx (hunk 2 above)
Comment (bene2k1) [inline labs/ipfs/api-cli/pin-operations.mdx:154]:
```suggestion
This operation allows you to delete a given pin within a volume using its pin ID.
```

**Pull request merged**